Src/BootCamp.Chapter/Logging/Log.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unnecessary 'using'.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5150047)